eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,56 @@
+$Language = "c"
+$Lang = "c"
+$PackageRepository = "C"
+$packagePattern = "*.json"
+$MetadataUri = ""
+
+function Extract-c-PkgProperties ($pkgPath, $serviceName, $pkgName)
+{
+  return null
```

Comment:
`$null`

Reply:
I'd suggest we just don't have this function in this case. In our common code we can check to see if this function exists and if it does call it otherwise handle it however we need to.